### PR TITLE
[aws-ints] Migrate VPC flow logs lambda to python3

### PR DESCRIPTION
**Pull request by doellerbe**
### What does this PR do?

This PR migrates the VPC flow logs lambda to python3. AWS announced EOL for python 2.7 is scheduled for July 15th, 2021

https://aws.amazon.com/blogs/compute/announcing-end-of-support-for-python-2-7-in-aws-lambda/

### Motivation
https://datadoghq.atlassian.net/browse/AWS-1124

### Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] Breaking change
- [ ] Misc (docs, refactoring, dependency upgrade, etc.)

### Check all that apply

- [ ] This PR's description is comprehensive
- [ ] This PR contains breaking changes that are documented in the description
- [ ] This PR introduces new APIs or parameters that are documented and unlikely to change in the foreseeable future
- [ ] This PR impacts documentation, and it has been updated (or a ticket has been logged)
- [ ] This PR's changes are covered by the automated tests
- [ ] This PR collects user input/sensitive content into Datadog
- [ ] This PR passes the integration tests (ask a Datadog member to run the tests)
- [ ] This PR passes the unit tests 
- [ ] This PR passes the installation tests (ask a Datadog member to run the tests)

### CODE REVIEW

**Comment by sabiurr** [comment]
Did we do any tests to make sure this is still working as expected?

**Comment by ashwinjv** [comment]
> Did we do any tests to make sure this is still working as expected?

This was tested by spinning up a new lambda forwarder with this and py3.7 runtime. `aws.vpc.flowlogs*` metrics are coming in from the new forwarder without any errors in the logs

File: aws/vpc_flow_log_monitoring/lambda_function.py [hunk 1]
```diff
@@ -23,27 +21,64 @@
...
+
+
+# Preload the keys so we can bail out early if they're misconfigured
 # Alternatively set datadog keys directly
 # datadog_keys = {
 #     "api_key": "abcd",
 #     "app_key": "efgh",
 # }
+datadog_keys = _datadog_keys()
+print('INFO Lambda function initialized, ready to send metrics')
```
Comment (tianchu) [inline aws/vpc_flow_log_monitoring/lambda_function.py]: Why not use python builtin logger?

File: aws/vpc_flow_log_monitoring/vpc-flow-log-sam-template.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ Resources:
       Policies:
         KMSDecryptPolicy:
           KeyId: !Ref KMSKeyId
-      Runtime: python2.7
+      Runtime: python3.7
```
Comment (tianchu) [inline aws/vpc_flow_log_monitoring/vpc-flow-log-sam-template.yaml:21]: Why not 3.8?

Comment (doellerbe) [reply aws/vpc_flow_log_monitoring/vpc-flow-log-sam-template.yaml:21]: I saw other lambdas on 3.7 and figured this was safe.

File: aws/vpc_flow_log_monitoring/lambda_function.py (hunk 1 above)
Comment (doellerbe) [reply aws/vpc_flow_log_monitoring/lambda_function.py]: No reason. I can change the prints to be info logs.

**Review by tianchu — APPROVED** [review]

**Pull request merged**